src/language_server.ts

```diff
@@ -259,6 +347,7 @@ export class LanguageServer {
 			};
 			return Promise.reject("failed to install LS");
 		} else{
+			log("LS installed with: "+ cmdArgs.join(' '));
```

Comment:
I like the additional logging but perhaps it would be better to log this right before running the command? This would cover the failure case as well.